src/hw/main.cpp

```diff
@@ -161,9 +219,10 @@ void createInventory(boost::asio::io_service& io,
                         manager.config.chassisSerial =
                             std::get<std::string>(value);
                     }
-                    else if (property.find("PRODUCT_INFO_AM") == 0)
+                    else if (property.find("_INFO_AM") != std::string::npos)
```

Comment:
It is not in the end, it like "PRODUCT_INFO_AMxxx", where xxx in number, staring with 1.

Reply:
@alatarum, he meant that searching for "_INFO_AM" from the end (rfind) may find the substring faster, because it is usually closer to the end than to the start of the string.